platform/os.go

```diff
@@ -4,8 +4,9 @@ import (
 	"bufio"
 	"fmt"
 	"io"
-	"log"
 	"os"
+
+	"github.com/Azure/azure-container-networking/log"
```

Comment:
Is this change required for zaplogger ?

Reply:
No, it's a bug here that it should use legacy log instead of go-buildin log "import log"; and this package is used by CNS only, so this does not need to be zap logged.